qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/PatientSteps.java

```diff
@@ -0,0 +1,262 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.PatientGenerator;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.helper.TestPatient;
+import org.openmrs.contrib.qaframework.page.DataManagementPage;
+import org.openmrs.contrib.qaframework.page.MergePatientsPage;
+import org.openmrs.contrib.qaframework.page.PatientVisitsDashboardPage;
+import org.openmrs.contrib.qaframework.page.RegistrationPage;
+
+public class PatientSteps extends Steps {
+	
+	private DataManagementPage dataManagementPage;
+	private MergePatientsPage mergePatientsPage;
+	private PatientVisitsDashboardPage VisitsDashboardPage;
```

Comment:
do you need to declare visitDashboardPage  variable?

Reply:
> do you need to declare visitDashboardPage variable?

@kdaud Actually not. The mistake I made was to call it VisitDashboardPage.